discord/enums.py

```diff
@@ -1078,6 +1080,15 @@ class SubscriptionStatus(Enum):
     inactive = 2
 
 
+class ThreadAutoArchiveDuration(enum.IntEnum):
```

Comment:
```suggestion
class ThreadArchiveDuration(enum.IntEnum):
```
Rename it here and then use `from ... import ThreadArchiveDuration as ThreadArchiveDurationEnum`